package.json

```diff
@@ -36,7 +36,7 @@
     "d3-dispatch": "^1.0.6",
     "d3-scale": "^3.2.1",
     "d3-scale-chromatic": "^1.5.0",
-    "d3-selection": "1.4.2",
+    "d3-selection": "^1.4.0",
```

Comment:
omg, i did not even realize this was a hard dependency.  It definitely should be a range `^`.  Must have been a typo and I wasn't able to notice it this whole time lol.

Is there a reason to prefer `1.4.0` over `1.4.2`? Asking to get advice when building packages if it's better to stay up-to-date on latest dependencies or to prefer older dependencies that are non-breaking (relying on semver and pointing to an old version).

Reply:
Ah, I've been misreading the docs at https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004, reading it round the wrong way, left-most semver digit gets maintained.  I'll make the change to `^1.4.2` and push again.  In any case, putting a caret here will put wordcloud on the same footing at `^1.1.0`, the dependency for `d3-transition`, if that was causing the conflict.

Reply:
And no, no reason to prefer the original version I had -- my mistake; my eyes are glazing over from looking at all these lockfiles! 🤣